content/4_Gold/Divis.mdx

```diff
@@ -51,6 +51,14 @@ If you've never encountered any **number theory** before, AoPS is a good place t
 	>
 		good book
 	</Resource>
+	<Resource
+		source="Aditya Khurmi@AoPS User: Wizard_32"
+		title="Modern Olympiad Number Theory"
+		url="https://drive.google.com/file/d/1BcJTLjQaelZ4w_70oHKyImC2I8zLfyrt/view"
```

Comment:
you should link to the AoPS post instead since it has a bit more info (https://artofproblemsolving.com/community/c6h2344755p18938156)

Reply:
wait, I got rid of it in the top table. Should I add it back?

Reply:
er I meant in all of the occurrences that remain

Reply:
but everything in the aops post is just people complementing the author. so there really isn't that much info

Reply:
I was referring to the errata

Reply:
oh ok. so should I just link the errata or the original post?

Reply:
the post

Reply:
k

Reply:
done